spec/ruby/core/symbol/all_symbols_spec.rb

```diff
@@ -1,14 +1,19 @@
 require_relative '../../spec_helper'
 
 describe "Symbol.all_symbols" do
-  it "returns an array containing all the Symbols in the symbol table" do
+  it "returns an array of Symbols" do
     all_symbols = Symbol.all_symbols
     all_symbols.should be_an_instance_of(Array)
-    all_symbols.all? { |s| s.is_a?(Symbol) ? true : (p s; false) }.should == true
+    all_symbols.all? { |s| s.is_a?(Symbol) }.should be_true
```

Comment:
Here it would be better to use:
```ruby
    all_symbols.each { |s| s.should be_an_instance_of(Symbol) }
```
That way, if it fails it will show what `s` is, instead of an unhelpful `expected true but was false`.

Reply:
Fixed.